src/interfaces/IAppProps.ts

```diff
@@ -0,0 +1 @@
+export default interface HelloProps { greet: string; company: string; }
```

Comment:
formatting is required! Readability matters! Please fix this.
```
export default interface HelloProps { 
        greet: string; 
        company: string; 
}
```

Reply:
Okay, I Will Do it, Thanks.